src/interface/src/app/features/features.json

```diff
@@ -1,5 +1,6 @@
 {
   "login": true,
+  "additional_layers": true,
```

Comment:
After some thought, I like hidden_ more than unimplemented_ since that is exactly what the code is doing.  I can see a reason for the future to keep using the flags for new layers that are only partially implemented, allowing us to hide them from the public (e.g. in prod), or to keep them hidden until we know what their future really will be (which is in the case for these layers).